rclcpp/include/rclcpp/utilities.hpp

```diff
@@ -26,7 +27,6 @@
 #include "rclcpp/visibility_control.hpp"
 
 #ifdef ANDROID
-#include <sstream>
```

Comment:
Why was this deleted?

Reply:
Leftover of when `join()` was here. Reverted.